src/project-2.js

```diff
@@ -3,6 +3,10 @@
 const getBiggest = (x, y) => {
   // x and y are integers.  Return the larger integer
   // if they are the same return either one
+  if (x > y) {
```

Comment:
I usually like to simplify `if` statements like this to look a little different:
```
if (x > y) return x;
return y;
```

You could also use a ternary operator:
```
return x > y ? x : y;
```

Which would simplify to ```const getBiggest = (x, y) => x > y ? x : y;```